.github/workflows/ingestion-functions-deploy.yml

```diff
@@ -2,7 +2,7 @@ name: Ingestion functions deploy
 
 on:
   push:
-    branches: [main]
+    branches: [main, 1.6-stable]
```

Comment:
I think `*-stable` would be better, as we'll probably change the version at some point :)